src/charts/utilization/utilization-chart.html

```diff
@@ -1,7 +1,7 @@
 <div class="utilization-chart-pf">
-    <span class="title">{{config.title}}</span>
+    <span class="h3">{{config.title}}</span>
```

Comment:
Yeah, I thought of doing that, but I wanted to inherit from patternfly styles so if h3 changes in patternfly then this changes as well.